### ISSUE TITLE
Regression: can't find strong's no., Blank link window... 

### ISSUE THREAD

**Issue by bvahedy** [AndBible/and-bible#2028]
It looks like the last build is not finding the right index to a strong's number. The link window is blank with a running circle. If I swipe left or right then the link window displays the next strong's no., If I tap on a different strong's no. It will show it but it's I go back to the previous no. It just displays the last no. 

This is a regression caused by some mod(s) from last build. Hopefully the log will show what is happening.
The no. I tap on is 2424:
![Screenshot_20220101-064647_Bible Study](https://user-images.githubusercontent.com/46405809/147853319-bdb990e3-46ed-428b-adf1-3d377e72b3cd.jpg)

**Comment by tuomas2** [comment]
which log?

**Comment by tuomas2** [comment]
https://support.andbible.org/scp/tickets.php?id=1180

**Comment by tuomas2** [comment]
can anyone reproduce? i can't.
thinking about severity vs release.

**Comment by bvahedy** [comment]
Sorry I thought the bug report produced a logcat.
I think more users are having this issue. When they tap on a strong's word
it displays the previous definition because it can't read or find the
correct strong's number.
Something happened to the way it's looking for a strong's no. I tried the
strongsReal Hebrew and Greek from the download and it seems to work fine
but the dictionary I had used for the last 8 years is now not working.
